bridges/snowbridge/pallets/inbound-queue-v2/src/lib.rs

```diff
@@ -208,6 +209,10 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type OperatingMode<T: Config> = StorageValue<_, BasicOperatingMode, ValueQuery>;
 
+	/// Keep track of tips added for a message as an additional relayer incentivization.
+	#[pallet::storage]
+	pub type Tips<T: Config> = StorageMap<_, Blake2_128Concat, u64, u128, ValueQuery>;
```

Comment:
Its useful to know whether there is actually a tip rather than a zero tip.

```suggestion
	pub type Tips<T: Config> = StorageMap<_, Blake2_128Concat, u64, u128, OptionQuery>;
```

Reply:
Addressed in https://github.com/vgeddes/polkadot-sdk/pull/23/commits/52404b229def2f307f44f552b61f3000adaf9cac.